tests/testthat/test-skip-vcr-off.R

```diff
@@ -0,0 +1,9 @@
+test_that("skip_if_vcr_off works when vcr on", {
+  expect_silent(skip_if_vcr_off())
+})
+
+test_that("skip_if_vcr_off works when vcr off", {
+  withr::local_envvar("VCR_TURN_OFF" = TRUE)
+  expect_error(skip_if_vcr_off(), class = "skip")
```

Comment:
So this is intended behavior when running tests for vcr, correct?

```r
── Skip (test-skip-vcr-off.R:7:3): skip_if_vcr_off works when vcr off ─
Reason: Not run when vcr is off
```

Reply:
ah maybe not https://github.com/r-lib/testthat/issues/1240